rollup.config.js

```diff
@@ -27,25 +29,43 @@ const componentPaths = readdirSync('components/', { withFileTypes: true })
 const iconsDirIndex = componentPaths.indexOf('icons');
 componentPaths.splice(iconsDirIndex, 1);
 
-export default componentPaths.map((component) => ({
-  input: `components/${component}/index.js`,
-  plugins: [
-    litcss({
-      include: '**/*.scss',
-      transform: transformSassToCss,
-      uglify: true,
-    }),
-  ],
-  output: {
-    dir: `components/${component}/`,
-    format: 'esm',
-    sourcemap: true,
+export default [
+  ...componentPaths.map((component) => ({
+    input: `components/${component}/index.js`,
+    plugins: [
+      litcss({
+        include: '**/*.scss',
+        transform: transformSassToCss,
+        uglify: true,
+      }),
+    ],
+    output: {
+      dir: `components/${component}/`,
+      format: 'esm',
+      sourcemap: true,
+    },
+    external: [
+      'tslib',
+      'lit',
+      /^lit\/.*/,
+      /^carbon-web-components\/.*/,
+      /^@carbon\/.*/,
+    ],
+  })),
+  {
+    input: `components/ui-shell/index.js`,
+    plugins: [
+      litcss({
+        include: '**/*.scss',
```

Comment:
As a comment maybe here we could add only the CSS related with the UI-Shell component to be even more specific. Anyway LGTM! 🎉

Reply:
As the input is `components/ui-shell/index.js`, I assume that it only find the ui-shell styles. But we can investigate it.